echo.go

```diff
@@ -67,6 +67,9 @@ type (
 	// Echo is the top-level framework instance.
 	Echo struct {
 		common
+		// startupMu is mutex to lock Echo instance access during server configuration and startup. Useful for to get
+		// listener address info (on which interface/port was listener binded) without having data races.
+		startupMu        sync.RWMutex
```

Comment:
Maybe use `startupMutex` or just `mutex` here.

Reply:
renamed to `startupMutex`